packages/dev/core/src/Cameras/arcRotateCamera.ts

```diff
@@ -963,6 +963,10 @@ export class ArcRotateCamera extends TargetCamera {
                         this._target.copyFrom(this._transformedDirection);
                     }
                 } else {
+                    if (this.parent) {
+                        const m = TmpVectors.Matrix[0];
+                        Vector3.TransformCoordinatesToRef(this._transformedDirection, this.parent.getWorldMatrix().getRotationMatrix().invertToRef(m), this._transformedDirection);

```

Comment:
getRotationMatrix generates a new matrix, would be better to use the getRotationMatrixToRef and then invert it in place (the invert function inverts in place).

Reply:
I get it.